files/models.py

```diff
@@ -26,8 +24,8 @@ class Inode(FYNode):
     Actual file/dir existing in filesystem
     """
     owner = models.ForeignKey(User)
-    created = models.DateTimeField(default=timezone.now)
-    updated = models.DateTimeField(default=timezone.now)
```

Comment:
oh, I see.
so
```python
 created = models.DateTimeField(default=timezone.now)
modified = models.DateTimeField(default=timezone.now)
```
And we have to change the value of `modified` whenever the `mtime` of the file object changes?